pykickstart/commands/sshpw.py

```diff
@@ -148,5 +184,11 @@ class F24_SshPw(F13_SshPw):
 
     def _getParser(self):
         op = F13_SshPw._getParser(self)
-        op.add_argument("--sshkey", action="store_true", default=False)
+#todo: this description doesn't make sence. Is this the same as the sshkey command
+# or different. Why is it a boolean flag instead of accepting the key content ?
```

Comment:
The sshpw command only affects the installation environment - it defines a user and ssh key that can be used while anaconda is running.  It has no effect on the finished system.  The sshkey command allows you to associate an ssh key with a user on the finished system.  That user could either already exist or be one created with the user command.

And then if you give --sshkey, the password you provided is interpreted to be the key.  So, "sshpw --username=A blah" means user A has the password of blah.  "sshpw --username=A --sshkey blah" means user A has an ssh key of blah, and no password.